### ISSUE TITLE
I wish  aliases 「 insert to editor」

### ISSUE THREAD

**Issue by Akiyam-hub** [tadashi-aikawa/obsidian-another-quick-switcher#245]
Thank you for always providing great features!

current, 「 insert to editor」 is only insert of  file name 
```
ex. [[filename]] or [filename](filename.md)
```

I wish to  insert aliases(or properties value)  「 insert to editor」 as option

it is useful to be a different thing from filename and aliases
```
ex. [aliases](filename.md)
```

**Comment by Akiyam-hub** [comment]
Thank you for your reply .

Your proposed behavior is exactly what I would like.

It would work even when there are multiple alias values.

I hope you will implement it!

**Comment by tadashi-aikawa** [comment]
@Akiyam-hub 
I have released version 11.6.0-beta for now. Since there are many optional patterns and the implementation has become complex, it seems likely that there are bugs. If there appear to be no issues with the operation, I plan to make an official release. If you are using BRAT, it would be helpful if you could try it out.

**Comment by Akiyam-hub** [comment]
Thank you very much !

I tried to install it as soon as possible.
currency, it works fine for my use.

**Comment by tadashi-aikawa** [comment]
I’m glad to hear that!

I will reopen this issue now and close it after the official release.

**Comment by tadashi-aikawa** [comment]
Thank you for your patience, @Akiyam-hub .

I have released in [v11.6.0](https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/releases/tag/11.6.0) 🚀
